tf-lambda/variables.tf

```diff
@@ -2,6 +2,14 @@ variable "target_env" {
   description = "AWS workload account env (e.g. dev, test, prod, sandbox, unclass)"
 }
 
+variable "dev_public_frontnd" {
```

Comment:
Let's use a single var for base_url_public_frontend, or something like that. We can differentiate between environments in the terraform call